gulpfile.mjs

```diff
@@ -1764,33 +1758,30 @@ gulp.task(
     setTestEnv,
     "generic",
     "components",
-    function runBotBrowserTest() {
-      return streamqueue(
```

Comment:
In this case using `streamqueue` was already unnecessary because there was only a single readable stream to begin with (note the difference with the `browsertest` target just above that already did it without `streamqueue`).